src/ComponentInstaller.php

```diff
@@ -458,11 +457,10 @@ private function marshalPackageComponents(array $extra, Collection $packageTypes
         $componentsByType = (new Collection($extra))
             ->filter(
             /**
-             * @psalm-suppress UnusedClosureParam https://github.com/vimeo/psalm/issues/8095
              * @param non-empty-string $typeIdentifier
              */
-                fn (array $components, string $typeIdentifier) => $supportedTypes->anySatisfies(
-                    fn (int $supportedType) => $supportedType === $packageTypes->getKey($typeIdentifier)
+                static fn (array $components, string $typeIdentifier) => $supportedTypes->anySatisfies(
```

Comment:
@Ocramius isn't `static` usage deprecated as of PHP 8.2 (or 8.3)?

Reply:
Not AFAIK